pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -255,6 +259,8 @@ func configFromEnv() (cfg Config, ok bool) {
 	return
 }
 
+// ReadConfig reads values from environment variables and the cloud.conf, prioritizing cloud-config
+// WARNING: depricated
```

Comment:
```suggestion
// WARNING: deprecated
```

Reply:
i think we should consider excluding or changing this message btw

Reply:
ok, i just took both of these out entirely